src/CoreFoundation/CFString.cs

```diff
@@ -115,14 +115,20 @@ public class CFString
 		[DllImport (Constants.CoreFoundationLibrary, CharSet=CharSet.Unicode)]
 		extern static IntPtr CFStringGetCharacters (IntPtr handle, CFRange range, IntPtr buffer);
 
-		internal static IntPtr LowLevelCreate (string str)
+		public static IntPtr CreateNative (string str)
```

Comment:
Not a big deal, but now that the API is public I think we should also make the parameter name nicer:

```suggestion
		public static IntPtr CreateNative (string value)
```

Reply:
hmm... I copied the exact signature from `NSString`
```csharp
public static IntPtr CreateNative (string str)
```
but I also find it nicer :)